examples/ZMQPP/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.2)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-ZMQPP)
+
+hunter_add_package(ZMQPP)
+
+find_package(ZMQPP CONFIG REQUIRED)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
> if(CMAKE_CXX_STANDARD LESS 11)

Toolchains add `-std=c++11` flag to `CMAKE_CXX_FLAGS` manually, `CMAKE_CXX_STANDARD` is not set. Using `CMAKE_CXX_STANDARD` require CMake 3.1+:
* https://cmake.org/cmake/help/v3.0/variable/CMAKE_CXX_STANDARD.html (not found)
* https://cmake.org/cmake/help/v3.1/variable/CMAKE_CXX_STANDARD.html (found)

Also this bug should be fixed first (when `CMAKE_CXX_STANDARD` set we need to add `-std=c++11` manually to custom schemes):
* https://github.com/ruslo/hunter/issues/322

However I think this is okay to have it in examples since last version of CMake used for testing/uploading (but still under `if(no_toolchain)` condition).

> in this case the user will get a correct error message

Usually such checks done in `hunter.cmake` file but I don't know how to do it nicely in this case. First problem is that we need to create module which read both `CMAKE_CXX_STANDARD` and `CMAKE_CXX_FLAGS`. Second problem is that when `hunter.cmake` loaded  `CMAKE_CXX_FLAGS` may be in modified state (i.e. may contain `-std=c++11` locally but not globally, well wrongly).

You can add check to C++ code. `__cplusplus` macro set to `201103L` for `c++11`.